crates/model/src/orders/market_if_touched.rs

```diff
@@ -524,6 +527,30 @@ impl From<OrderInitialized> for MarketIfTouchedOrder {
     }
 }
 
+impl Display for MarketIfTouchedOrder {
```

Comment:
This one just needs the current `key=value, ` convention.

Reply:
fixed